src/ripple/server/impl/ServerHandlerImp.cpp

```diff
@@ -200,12 +200,33 @@ ServerHandlerImp::onStopped (HTTP::Server&)
 
 // Run as a couroutine.
 void
-ServerHandlerImp::processSession (std::shared_ptr<HTTP::Session> const& session,
+ServerHandlerImp::processSession (std::shared_ptr<Session> const& session,
     std::shared_ptr<JobCoro> jobCoro)
 {
+    auto forwarded_for =
```

Comment:
Nits: `forwarded_for` -> `forwardedFor` and it could be `auto const`. The `user` var below can also be `auto const`.